ruby-plus-devkit27/plan.ps1

```diff
@@ -0,0 +1,51 @@
+$pkg_name="ruby-plus-devkit27"
```

Comment:
@TheLunaticScripter Looking at this again, should the package name be `ruby27-plus-devkit` maybe? 🤦

Reply:
Yeah that makes sense since it's the ruby that's 2.7 not the devkit part. I'll change that.